src/app/api/generate/route.ts

```
@@ -1,91 +1,150 @@
+          prompt: data?.prompt,
+          imageCount: data?.imageCount,
+          aspectRatio: data?.aspectRatio,
         });
+        break;
+      case 'black-forest-labs/flux-schnell':
+        break;
+      case 'black-forest-labs/flux-dev':
+        break;
+      default:
+        throw new Error('No image model found');
+        break;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)